.circleci/config.yml

```diff
@@ -3,8 +3,48 @@ version: 2
 
 references:
   circleci_docker_primary: &circleci_docker_primary trussworks/circleci-docker-primary:tf12-5ee2cb35bc081dbc91da46afc3904ef0e1c111fb
+  postgres: &postgres postgres:11.6
+  db_migrate: &db_migrate ${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:latest
 jobs:
-  pre_test:
+  build_db_migrate_image:
+    docker:
+      - image: *circleci_docker_primary
+        environment:
+          AWS_DEFAULT_REGION: us-west-2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build the flyway migrations for testing
+          command: |
+            ./scripts/build_db_image "Dockerfile.db_migrations" "easi-db-migrate" "latest"
+
+  test:
+    docker:
+      - image: *circleci_docker_primary
+        environment:
+          PG_SSL_MODE: disable
+          PGUSER: postgres
```

Comment:
Can we align `PGUSER` and `POSTGRES_USER` key names below for consistency?  If I'm reading this correctly `POSTGRES_USER` is what the postgres container uses and we made up `PGUSER`. Is there any reason for them to be different other than it got written that way first?

Reply:
`PGUSER`, `PGPASS`, `PGHOST` etc is the format specified by [libpq](https://www.postgresql.org/docs/current/libpq-envars.html) for creating connections. Many (but not all) postgres client tools follow this pattern. The postgres [docker image](https://hub.docker.com/_/postgres/) uses different variable names for instantiating the database in a new container. In particular, `POSTGRES_PASSWORD` is mandatory. It appears `PGUSER` is being read in `store_test.go`. 

Since the auth method is "trust", `PGPASS` is not relevant. I will remove it.

Reply:
Hm, I may be wrong about `POSTGRES_PASSWORD`. I'll experiment a bit more with which of these vars are actually needed in CI.